apps/server/src/modules/lesson/uc/lesson.uc.spec.ts

```diff
@@ -193,4 +193,48 @@ describe('LessonUC', () => {
 			});
 		});
 	});
+
+	describe('getTasks', () => {
```

Comment:
add please some empty lines between expect and variables pin the it blocks

Reply:
Done